### ISSUE TITLE
Slow builds with high number of unchanged Localizables

### ISSUE THREAD

**Issue by Dschee** [Flinesoft/BartyCrouch#9]
When using the full automatic build style builds with unchanged IB files can take 5-10 seconds longer than without BartyCrouch even when nothing is changed with many files (say thousand translation target files) to check. This should be fixed, for example by using a build artifact file that keeps track of the last time BartyCrouch was run and skip files that didn't change since then.

**Comment by Dschee** [comment]
I investigated this further also adding a performance test for future tracking in cb8235ecc479f1a032d84f19fbbe9f624ab96675.

The problem in my project seemed to be that I ran `bartycrouch $PROJECT_PATH` which also included the .git folder where the entire history of my project lied within. Changing my script to `bartycrouch "$PROJECT_PATH"` like suggested in 4f041b727f351ce42cb104235ccb29da401083b6 already fixed the issue (somehow) but to be sure I've updated the README in c1b1930311b20e5c04ebc77337e68c708f41745a and 80b3f6f7649c79356805360b24871e067edc6108 to explicitly state performance drawbacks when configured wrongly.
